cdap-watchdog/src/main/java/co/cask/cdap/logging/LoggingConfiguration.java

```diff
@@ -59,7 +60,7 @@
 
   public static final long DEFAULT_LOG_SAVER_EVENT_BUCKET_INTERVAL_MS = 1 * 1000;
   public static final long DEFAULT_LOG_SAVER_MAXIMUM_INMEMORY_EVENT_BUCKETS = 4;
-  public static final long DEFAULT_LOG_SAVER_INACTIVE_FILE_INTERVAL_MS = 60 * 60 * 1000;
+  public static final long DEFAULT_LOG_SAVER_MAX_FILE_LIFETIME_MS = TimeUnit.HOURS.toMillis(1);
```

Comment:
Since it only matters with impersonation turned on, I think we should just have documentation on limitation or requirements when using impersonation.
Filed a JIRA for documenting that later: https://issues.cask.co/browse/CDAP-7118